### ISSUE TITLE
Tests fail with non-English locale

### ISSUE THREAD

**Issue by eserte** [gnustavo/SVN-Hooks#6]
If a locale-capable svn is available and a non-English locale is in effect (e.g. by setting LC_ALL=de_DE.UTF-8 on freebsd systems), then the test suite may fail:
```
Couldn't grok version from 'svn --version' command at t/test-functions.pl line 37.
# Looks like you planned 13 tests but ran 10.
# Looks like your test exited with 255 just after 10.
t/02-generic.t .............. 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 3/13 subtests 
```

**Comment by eserte** [comment]
I see that LC_MESSAGES=C is set for testing --- unfortunately LC_ALL has precedence over LC_MESSAGES, so you want to set it, too.

And I just verified that the problem also happens on linux systems, not just freebsd systems.

**Comment by gnustavo** [comment]
Thank you, Slaven. It's fixed on cfd6e4e0. I just released version 1.30 which should appear shortly on CPAN.
